tests/test_basic.py

```diff
@@ -1116,6 +1116,25 @@ def test_static_files():
     rv.close()
 
 
+def test_static_path_depricated():
```

Comment:
Thanks for fixing that. Were you able to fix it or do I need to fix and push again?